src/Cli/Program.cs

```diff
@@ -29,6 +29,8 @@ public static int Main(string[] args)
                 }
             );
 
+            DotNetEnv.Env.TraversePath().Load();
```

Comment:
Usage of this library in the current implementation, overwrites environment variables on the system yes?
The PR description:

> Now, users can maintain a local file .env to manage environment variables instead of setting environment variables in the system.
> If .env file is not used, it will take precedence over system environment variables.

should clarify that users no longer manually have to modify env vars, because this package does it for them.
Saying that there are "precedence" rules seems like we are saying existing environment variables are not touched, but if the env var is defined in the .env file, it will be overwritten.

Reply:
we do not update the environment variables defined in the system.